core/plugins/org.polarsys.capella.core.af.integration/src/org/polarsys/capella/core/af/integration/CapellaMetadataProvider.java

```diff
@@ -185,22 +185,24 @@ public IStatus checkMetadata(URI sessionResourceURI, ResourceSet set) {
 
   /**
    * Returns whether the file of the given fileVersion requires a migration to be compatible to currentVersion
+   * 
+   * @see See also org.polarsys.capella.core.data.migration.af.ViewpointMigrationContribution.isMigrationPossible
    */
   private IStatus isMigrationRequired(Version fileVersion, Version currentVersion) {
-
+    
+    // If model from <MAJOR>.x towards <MAJOR>.y, we requires a migration.
+    if (fileVersion.getMajor() == currentVersion.getMajor() && fileVersion.getMinor() != currentVersion.getMinor()) {
+      return new Status(IStatus.ERROR, AFIntegrationPlugin.getSymbolicName(),
+          NLS.bind(Messages.WrongCapellaVersionException_DetailedMessage, fileVersion));
+    }
+    
     // if not the same major/minor we requires a migration
     if (!(fileVersion.getMajor() == currentVersion.getMajor() && fileVersion.getMinor() == currentVersion.getMinor())) {
       return new Status(IStatus.ERROR, AFIntegrationPlugin.getSymbolicName(),
           Messages.WrongCapellaVersionException_Message);
     }
 
-    // if model from 1.4.0 towards 1.3.x, we requires a migration
-    if (fileVersion.getMajor() == 1 && fileVersion.getMinor() == 3 && fileVersion.getMicro() == 0) {
```

Comment:
is it intended to not have this requirement anymore?

Reply:
yes, this is dead code.
it checks if the model is from 1.3.0 and current version is 1.3.x. (which is quite old as there is 1.4.x, 5.x and 6.0 since then 😋